src/main/java/VASSAL/build/module/map/KeyBufferer.java

```diff
@@ -147,8 +160,19 @@ else if (!s.isExpanded()) {
       map.getPieceCollection().moveToFront(to_front);
     }
     else {
+      bandSelect = BandSelectType.NORMAL; //BR// Allowed to band-select  
+    }
+    
+    if (bandSelect != BandSelectType.NONE) {
+      bandSelectPiece = null;
       if (!e.isShiftDown() && !e.isControlDown()) { // No deselect if shift key down
         kbuf.clear();
+        
+        //BR// This section allows band-select to be attempted from non-moving pieces w/o preventing click-to-select from working 
+        if ((bandSelect == BandSelectType.SPECIAL) && (p != null) && !ignoreEvent) {
```

Comment:
Boolean operators have lower precedence than the comparison operators, so you don't need the extra parentheses here.

Reply:
It's an old habit of mine. I personally find it more readable but am happy to "conform" if it's bothering you.

Reply:
> Sure, you want me to make you one real quick? Or just point you in the direction of one where you could then change the does-not-stack parameters in something and see it work?

No need to make one if there's one I can just download (or might already have handy).

Reply:
If you pull down Paths of Glory 9.7, there are invisible pieces over every empty space. (See London or Berlin an example, many more).
1. First note that you can now start a band-select inside the boundaries of one of these without having the whole mouse stroke be "eaten" (you can try the same thing unpatched to see what I mean).
2. Then also notice that you nonetheless CAN band select these empty spaces (but why would you want to)
3. So then you can edit and go to the prototype "Empty Space" which is conveniently like the 3rd or 4th down the list in the prototypes. So you can edit the Does Not Stack for this, and use the new band select thing to make these things NOT band-selectable.
4. Now run the module and note that not only do the band selects not get "eaten" by the immobile piece being there, but that now these unwanted immobile pieces are filtered out of your resulting band selection (only other pieces remain in it).
5. And then if you want you can try the "Alt Key Only" version.

Reply:
Huh, I think I prefer the original map

Reply:
Haha :) Good news, it's still there. One of my big projects in "rescuing" 9.0 was putting back the Classic Map and getting it working right too.

Reply:
This looks like it works. Thanks.

Reply:
I got PoG almost immediately after it was first published... The "Delux" map just looks wrong to me.

Reply:
Yeah it was a major priority for the community to get both maps back working so that you could still play either the classic scenario or the "tournament" historical scenario still on the original map. I actually like both of the maps, but I didn't approve of only supporting one of them.